tools/syn2mas/src/migrate.mts

```diff
@@ -119,9 +119,9 @@ export async function migrate(): Promise<void> {
     fatals += 1;
   }
 
-  function makeUuid<T>(): UUID<T> {
-    return id128.Uuid4.fromRaw(
-      id128.UlidMonotonic.generate().toRaw(),
```

Comment:
I switched from UlidMonotonic because:

 - there is a very low chance of collision on the same timestamp with the random part 
 - UlidMonotonic is buggy when going "back in time"